railsbridge/docs
# Move localization from directory-based to i18n-based

This commit removes the need of having an specific folder per language
inside /docs

With only one top-level doc, we can handle translations on their
corresponding files now living under `locales`

Each language now has its own folder, which is going to make easier to
implement continuous localization with Crowdin

A quick way to test this change is:

* Run app locally with `bundle exec rake run`
* Go to `localhost:9292/installfest`. It should be in English by default
* Add `?locale=es` to `localhost:9292/installfest`
* Page will be in Spanish now using translated keys from yml files
* Replace `?locale=es` with `?locale=zh-tw`
* Page with be in Chinese now

See gif below:
![locales](https://user-images.githubusercontent.com/1335038/161322862-3690bfa4-c24f-4282-91f9-cb90124f6c3b.gif)
